torchinductor/codegen/triton.py

```diff
@@ -1105,14 +1116,18 @@ def end_current_reduction_loop():
     def is_contiguous(node):
         if node in (EnableReduction, DisableReduction):
             return True
-        return all(
-            dep.is_contiguous()
-            for dep in itertools.chain(node.read_writes.reads, node.read_writes.writes)
-        )
+        if node.is_reduction():
+            return all(
+                dep.is_contiguous()
+                for dep in itertools.chain(
+                    node.read_writes.reads, node.read_writes.writes
+                )
+            )
+        else:
+            return True
```

Comment:
Won't this give the wrong hint if we start passing hints to tiled pointwise?

Reply:
It would, but currently pointwise heuristics doesn't accept any arguments at all. We can name this function differently.